packages/dotcom-ui-footer/bower.json

```diff
@@ -1,7 +1,10 @@
 {
   "name": "dotcom-ui-footer",
   "dependencies": {
-    "o-footer": "^7.0.0"
+    "o-footer": "9.0.0-bower"
```

Comment:
```suggestion
    "o-footer": "^9.0.0-bower"
```
**nitpick:** We probably want to use a [caret range](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept) so we can pick up any minor releases to `o-footer`

Reply:
Unfortunately, as o-footer isn't pushed to bower by default any more, we need to fix the version here. Otherwise `^9.0.0-bower` resolves to `9.0.0` and caused dependency issues due to origami moving to `npm`

Reply:
Aaah that will be why I was having issues with storybook then! Looks to be fixed now, thanks.